### PR TITLE
Add testgrid tabs for dot release jobs

### PR DESCRIPTION
**Pull request by chaodaiG**
I believe missing tabs has caused testgrid backend failed to process

/assign @chizhg

### CODE REVIEW

File: config/prow/testgrid/testgrid.yaml [hunk 1]
```diff
@@ -975,18 +975,54 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: "knative-productivity-dev@googlegroups.com"
     num_failures_to_alert: 3
+  - name: serving-continuous
```
Comment (chizhg) [inline config/prow/testgrid/testgrid.yaml]: This one needs to be removed.

File: config/prow/testgrid/testgrid.yaml [hunk 2]
```diff
@@ -975,18 +975,54 @@ dashboards:
...
+    test_group_name: ci-knative-serving-continuous-0.10-continuous
+    base_options: "sort-by-name="
+    alert_options:
+      alert_mail_to_addresses: "knative-productivity-dev@googlegroups.com"
+    num_failures_to_alert: 3
+  - name: serving-dot-release
+    test_group_name: ci-knative-serving-dot-release-0.10-continuous
```
Comment (chizhg) [inline config/prow/testgrid/testgrid.yaml]: The test group name should be `ci-knative-serving-0.11-dot-release`.

**Comment by chaodaiG** [comment]
finally figured out how to do it, thanks!

File: config/prow/testgrid/testgrid.yaml [hunk 3]
```diff
@@ -969,149 +969,245 @@ dashboards:
...
     alert_options:
       alert_mail_to_addresses: "knative-productivity-dev@googlegroups.com"
     num_failures_to_alert: 3
-  - name: eventing
+  - name: serving-dot-release
+    test_group_name: ci-knative-serving-0.10-dot-release
+    base_options: "sort-by-name="
+    alert_options:
+      alert_mail_to_addresses: "knative-productivity-dev@googlegroups.com"
+    num_failures_to_alert: 3
```
Comment (chizhg) [inline config/prow/testgrid/testgrid.yaml]: Could you change this number to `1`? Our current strategy is sending alert for every release errors and 3 continuous test errors

Comment (chaodaiG) [reply config/prow/testgrid/testgrid.yaml]: I don't think it matters as it occurs very infrequently, would rather not make any further "optimization" in code

Comment (chizhg) [reply config/prow/testgrid/testgrid.yaml]: I see, makes sense.

**Pull request merged**